fe/fe-core/src/main/java/com/starrocks/sql/analyzer/DataCacheStmtAnalyzer.java

```diff
@@ -154,13 +154,15 @@ public Void visitDataCacheSelectStatement(DataCacheSelectStatement statement, Co
             // Duration for cache remains active.
             // Use PT0M to prevent expiration of the rule.
             // Use duration specified in ISO-8601 duration format (PnDTnHnMn).
-            long ttlSeconds = 0L;
-            try {
-                ttlSeconds = Duration.parse(properties.getOrDefault("ttl", "PT0M")).toSeconds();
-            } catch (DateTimeParseException e) {
-                throw new SemanticException(String.format(
-                        "Illegal ttl format, use duration specified in ISO-8601 duration format (PnDTnHnMn). Error msg: %s",
-                        e.getMessage()));
+            long ttlSeconds = context.getSessionVariable().getDatacacheTTLSeconds();
+            if (properties.containsKey("ttl")) {
```

Comment:
If the ttl only tacks effect for `cache select` population, I think the session variable is also unnecessary, because users can config it in `cache select`. This session makes users confused that all cached data, including populated by query, will be affected by the TTL.

Reply:
Some users will active warmup datacache with `cache select`,  However, other users only passively populate the cache with query. So it is necessary to session variable